plugin/pkg/managedseed/validator/admission.go

```diff
@@ -523,17 +532,14 @@ func (v *ManagedSeed) getSeedDNSProviderForCustomDomain(shoot *gardencorev1beta1
 			}
 			return nil, apierrors.NewInternalError(fmt.Errorf("could not get secret binding %s/%s: %v", shoot.Namespace, *shoot.Spec.SecretBindingName, err))
```

Comment:
Nit: Looks like an overlook from a previous PR. Can you please add this to your PR
```suggestion
			return nil, apierrors.NewInternalError(fmt.Errorf("could not get credentials binding %s/%s: %v", shoot.Namespace, *shoot.Spec.CredentialsBindingName, err))
```

Reply:
I have already proposed the change in https://github.com/gardener/gardener/pull/13552/changes#diff-700c464741e5c824302257f5f7ed3826e287424b2b8fc11d94808336faa684e8R535, so let's get it in with that PR instead, does this work for you?

Reply:
I forgot about that 🙈 Thanks for clarifying, works for me